kick/http.py

```diff
@@ -4,12 +4,13 @@
 import json
 import logging
 from typing import TYPE_CHECKING, Any, Coroutine, Optional, TypeVar, Union
+from curl_cffi import requests as requests_ORI
```

Comment:
There is no reason to change the name its imported under. Importing `curl_cffi` itself is better than changing the name of something from the library